packages/paste-core/primitives/box/package.json

```diff
@@ -25,7 +25,7 @@
     "tsc": "tsc"
   },
   "dependencies": {
-    "csstype": "^2.6.10",
+    "csstype": "^3.0.11",
```

Comment:
💡 If we polish up the [`@twilio-paste/style-props` package](https://github.com/twilio-labs/paste/pull/2376/files#diff-9ac9bd1116c3343d4b202132b364d1d77f4c53797b7f7e47ae1dc42ed46550a4R26), we should be able to list that as a dependency here, and not have to directly require `csstype`.